### PR TITLE
Add "Bring your own backend" section

### PR DESCRIPTION
**Pull request by svrnm**
Just a small addition to the README.md that explains how people can quickly make the demo work with their own backend.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,23 @@ docker compose up
...
+exporters:
+  jaeger:
+    endpoint: "jaeger:14250"
+    insecure: true
+  logging:
+  otlp:
+    endpoint: <your-endpoint-url>
+```
+
+When you now start the dem by running `docker compose up` you should see the traces flowing into your backend as well.
```
Comment (cijothomas) [inline README.md:60]:
```suggestion
When you now start the demo by running `docker compose up` you should see the traces flowing into your backend as well.
```

File: README.md [hunk 2]
```diff
@@ -43,6 +43,23 @@ docker compose up
 
 - And the Jaeger UI at: <http://localhost:16686>
 
+### Bring your own backend
+
+Likely you want to use the Webstore as a demo application for an observability backend you already have (e.g. an existing instance of jaeger, zipkin or one of the [vendor of your choice](https://opentelemetry.io/vendors/). Adding this backend is easy: Before running docker compose, open the [otelccol-config.yml](./src/otelcollector/otelcol-config.yml) with an editor and add an trace exporter for your backend, e.g.:
```
Comment (cijothomas) [inline README.md:48]: maybe - make it clear that this is only applicable if the vendor has native OTLP ingesting endpoint.
And mention that, some vendor backends requires separate exporters, available in collector-contrib repos.

Comment (svrnm) [reply README.md]: makes sense, thank you. Updated!

**Comment by svrnm** [comment]
@cijothomas updated based on your suggestions

File: README.md [hunk 3]
```diff
@@ -43,6 +43,25 @@ docker compose up
 
 - And the Jaeger UI at: <http://localhost:16686>
 
+### Bring your own backend
+
+Likely you want to use the Webstore as a demo application for an observability backend you already have (e.g. an existing instance of jaeger, zipkin or one of the [vendor of your choice](https://opentelemetry.io/vendors/). Adding this backend is easy: Before running docker compose, open the [otelccol-config.yml](./src/otelcollector/otelcol-config.yml) with an editor and add an trace exporter for your backend. For example, if your backend supports otlp extend the `exporters` section by the last 2 lines:
+
+```yaml
+exporters:
```
Comment (cijothomas) [inline README.md:51]: defining exporter alone is not sufficient. this must be added to the pipeline as well.

File: README.md [hunk 4]
```diff
@@ -43,6 +43,25 @@ docker compose up
...
+  jaeger:
+    endpoint: "jaeger:14250"
+    insecure: true
+  logging:
+  otlp:
+    endpoint: <your-endpoint-url>
+```
+
+Vendor backends might require you to add  additional parameters for authentication, please check their documentation. Some backends require different exporters, you will find them and their documentation available at [opentelemetry-collector-contrib/exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter).
```
Comment (cijothomas) [inline README.md:60]:
```suggestion
Vendor backends might require you to add  additional parameters for authentication, please check their documentation. Some backends require different exporters, you may find them and their documentation available at [opentelemetry-collector-contrib/exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter).  
```

File: README.md (hunk 3 above)
Comment (svrnm) [reply README.md:53]: I always forget the pipeline 😂 ... updated.

File: README.md [hunk 5]
```diff
@@ -43,6 +43,37 @@ docker compose up
...
+```yaml
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging, jaeger, otlp]
+```
+
+Vendor backends might require you to add  additional parameters for authentication, please check their documentation. Some backends require different exporters, you may find them and their documentation available at [opentelemetry-collector-contrib/exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter).
```
Comment (cijothomas) [inline README.md:72]:
```suggestion
Vendor backends might require you to add additional parameters for authentication, please check their documentation. Some backends require different exporters, you may find them and their documentation available at [opentelemetry-collector-contrib/exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter).  
```

**Review by cijothomas — APPROVED** [review]

**Comment by cartersocha** [comment]
Non blocking but do we want to document or produce a collector-contrib version? I couldn’t quite get the contrib image to work on our current set up last night

**Comment by svrnm** [comment]
> Markdown errors need to be fixed but lgtm.
> 
> If you’re not using them the rewrap & markdownlint vs code extensions are super helpful for this.

Did the edits in githubs editor, that of course does not help with that, went over to vscode and fixed that

**Review by reyang — APPROVED** [review]
LGTM.

**Pull request merged**